scripts/kind-e2e/e2e.sh

```diff
@@ -87,10 +87,46 @@ function kind_import_images() {
         kind --name cluster${i} load docker-image submariner:local
         kind --name cluster${i} load docker-image submariner-route-agent:local
     done
+
+    if [ ! -z "${global_CIDRs[cluster1]}" ]
+    then
+        docker tag quay.io/submariner/submariner-globalnet:$VERSION submariner-globalnet:local
+        for i in 1 2 3; do
+            echo "Loading globalnet image into cluster${i}..."
+            kind --name cluster${i} load docker-image submariner-globalnet:local
+        done
+    fi
+}
+
+function get_globalip() {
+    svcname=$1
+    gip=
+    attempt_counter=0
+    max_attempts=5
```

Comment:
Will 5 sec be enough all the time...  it wouldn't hurt to give this plenty more cushion, say 60.